clientlibrary/worker/worker.go

```diff
@@ -194,6 +203,15 @@ func (w *Worker) initialize() error {
 		log.Infof("Use custom checkpointer implementation.")
 	}
 
+	if w.kclConfig.EnableLeaseStealing {
+		if w.leasestealer == nil {
+			log.Infof("Creating DynamoDB based leasestealer")
+			w.leasestealer = chk.NewDynamoLeasestealer(w.kclConfig, w.checkpointer)
```

Comment:
No need to create lease stealer. It should be part of checkpointer but just disabled by default.

Reply:
Fixed